### PR TITLE
df, exec, exit, export: fix more info link

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/df.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # df
 
 > Verschafft einen Überblick über verfügbaren Speicherplatz im Dateisystem.
-> Weitere Informationen: <https://www.gnu.org/software/coreutils/df>.
+> Weitere Informationen: <https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/df.html>.
```
Comment (vitorhcl) [inline pages.de/common/df.md:4]:
```suggestion
> Weitere Informationen: <https://manned.org/df.1posix>.
```

File: pages.es/common/df.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # df
 
 > Entrega información general del uso de espacio en disco del sistema de archivos.
-> Más información: <https://www.gnu.org/software/coreutils/df>.
+> Más información: <https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/df.html>.
```
Comment (vitorhcl) [inline pages.es/common/df.md:4]:
```suggestion
> Más información: <https://manned.org/df.1posix>.
```

File: pages.fa/common/df.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # df
 
 > یک نمای کلی از میزان استفاده فضای دیسک و سیستم فایل ارائه می‌دهد.
-> اطلاعات بیشتر: <https://www.gnu.org/software/coreutils/df>.
+> اطلاعات بیشتر: <https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/df.html>.
```
Comment (vitorhcl) [inline pages.fa/common/df.md:4]:
```suggestion
> اطلاعات بیشتر: <https://manned.org/df.1posix>.
```

File: pages.fr/common/df.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # df
 
 > Montre un aperçu de l'utilisation de l'espace disque.
-> Plus d'informations : <https://www.gnu.org/software/coreutils/df>.
+> Plus d'informations : <https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/df.html>.
```
Comment (vitorhcl) [inline pages.fr/common/df.md:4]:
```suggestion
> Plus d'informations : <https://manned.org/df.1posix>.
```

File: pages.ta/common/df.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # df
 
 > கோப்பு முறைமை வட்டு இட உபயோகத்தின் மேலோட்டத்தை அளிக்கிறது.
-> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/df>.
+> மேலும் விவரத்திற்கு: <https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/df.html>.
```
Comment (vitorhcl) [inline pages.ta/common/df.md:4]:
```suggestion
> மேலும் விவரத்திற்கு: <https://manned.org/df.1posix>.
```

File: pages.zh/common/df.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # df
 
 > 提供文件系统磁盘空间使用情况的概览。
-> 更多信息：<https://www.gnu.org/software/coreutils/df>.
+> 更多信息：<https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/df.html>.
```
Comment (vitorhcl) [inline pages.zh/common/df.md:4]:
```suggestion
> 更多信息：<https://manned.org/df.1posix>.
```

File: pages.zh_TW/common/df.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # df
 
 > 概述檔案系統的磁碟空間的使用情況。
-> 更多資訊：<https://www.gnu.org/software/coreutils/df>.
+> 更多資訊：<https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/df.html>.
```
Comment (vitorhcl) [inline pages.zh_TW/common/df.md:4]:
```suggestion
> 更多資訊：<https://manned.org/df.1posix>.
```

**Review by vitorhcl — DISMISSED** [review]
We agreed on #11965 to use manned.org instead of pubs.opengroup.org

**Review by MrMw3 — APPROVED** [review]

File: pages.fr/common/df.md (hunk 4 above)
Comment (tricantivu) [reply pages.fr/common/df.md:4]: The space character after `Plus d'information` should be remove as well, right?

**Review by kbdharun — APPROVED** [review]

**Pull request merged**